src/Commands/Claw/ClawSmartRollers.h

```diff
@@ -1,18 +1,18 @@
 #ifndef CLAWSMARTROLLERS_H
 #define CLAWSMARTROLLERS_H
 
-#include "CommandBase.h"
+#include "Robot.h"
 #include "Subsystems/Claw.h"
 
 namespace tator {
 
-class ClawSmartRollers: public CommandBase {
+class ClawSmartRollers: public RobotCommand {
 public:
 	ClawSmartRollers(std::string name, YAML::Node config) :
-			CommandBase(name) {
+			RobotCommand(name) {
 		waitTime = config["waitTime"].as<double>();
 		isWaiting = false;
-		Requires(claw);
+		Requires(&*robot->claw);
```

Comment:
Oh lawd. The expression `&*robot->claw` made me sick. Why does it need to be like this?